web/server.js

```diff
@@ -1 +1,51 @@
-comming soon
\ No newline at end of file
+var express = require('express');
+var app = express();
+var bodyparser = require('body-parser');
+var fs = require('fs');
+var open = require('open');
+var exec_cmd = require('./exec_cmd');
+
+app.set('view engine', 'ejs');
+app.use(express.static('public'));
+app.use(bodyparser.urlencoded({extended : false}));
+
+var cmd_file = 'cmd.txt';
+const port = 3000;
```

Comment:
Why cmd_file is defined using **var** keyword and port is defined with **const** keyword?